toolbox/cijoe-pkg-xnvme/src/cijoe/xnvme/tests/logic/test_xnvme_tests_ioworker.py

```diff
@@ -0,0 +1,32 @@
+"""
+New addition
+"""
+import pytest
+from cijoe.xnvme.tests.conftest import xnvme_device_driver as device
+from cijoe.xnvme.tests.conftest import xnvme_setup
+
+
+@pytest.mark.parametrize(
+    "device,be_opts,cli_args",
+    xnvme_setup(labels=["dev"], opts=["be", "sync", "async", "admin"]),
+    indirect=["device"],
+)
+def test_verify(cijoe, device, be_opts, cli_args):
```

Comment:
This needs to be expanded with a `test_verify_sync` that replaces the `xnvme_tests_ioworker_sync.sh` from `next`